gnowsys-ndf/gnowsys_ndf/ndf/templates/ndf/footer1.html

```diff
@@ -0,0 +1,52 @@
+{% load i18n %}
+{% load get_latest_git_hash from simple_filters %}
```

Comment:
Can you explain why this footer1.html template is added? We are already using footer.html for footer